app/api/connectors/BaseDownstreamConnector.scala

```diff
@@ -69,6 +79,16 @@ trait BaseDownstreamConnector extends Logging {
     http.put(url(downstreamUri)).withBody(Json.toJson(body)).execute
   }
 
+  def putEmpty[Resp](downstreamUri: DownstreamUri[Resp])(implicit
```

Comment:
For which endpoint do we need a PUT with no request body?

Reply:
There was talk of making the restore a PUT. But this is for completeness.